diff -Nurwa dhcdbd-2.8-orig/src/dhcdbd.c dhcdbd-2.8/src/dhcdbd.c
--- dhcdbd-2.8-orig/src/dhcdbd.c	2006-08-25 21:45:05.000000000 +0200
+++ dhcdbd-2.8/src/dhcdbd.c	2007-10-28 16:08:59.388530456 +0100
@@ -924,11 +924,13 @@
                         return -1;
                 sprintf (dhc_args[arg], "dhc_dbus=%-2u", d_if->dhc_mode);
 
+#if 0
 #if DHCLIENT_EXTENDED_OPTION_ENVIRONMENT > 0
                 if ((dhc_args[++arg] = (char *) malloc (3)) == 0L)
                         return -1;
                 strcpy (dhc_args[arg], "-x");
 #endif
+#endif
 
                 if ((dhc_args[++arg] = (char *) malloc (3)) == 0L)
                         return -1;
